### PR TITLE
Homework11

### PR DESCRIPTION
**Pull request by MaksServatyak**

### CODE REVIEW

File: src/Lesson11/Task2.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package Lesson11;
+
+import java.time.LocalDateTime;
+
+public class Task2 {
```
Comment (Volodymyr95) [inline src/Lesson11/Task2.java]: What does it mean Task2?

File: src/Lesson11/Task1.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package Lesson11;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Task1 {
```
Comment (Volodymyr95) [inline src/Lesson11/Task1.java]: What does it mean Task1?

File: src/Lesson11/Task5.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package Lesson11;
+
+public class Task5 {
+    String text;
+
+    public void isPalindrom(String text) {
+        this.text = text;
+        int k=0;
+
+        for (int i = 0; text.length() > i; i++) {
```
Comment (Volodymyr95) [inline src/Lesson11/Task5.java]: solution from google

File: src/Lesson11/Task5.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            if (text.length() - 1 - i > i) {
+                k++;
+
+                if (text.charAt(text.length() - 1 - i) != text.charAt(i)) {
+                    System.out.println("не є поліндромом");
```
Comment (Volodymyr95) [inline src/Lesson11/Task5.java]: I 💓 but don't understand the Ukrainian language, if u have some problem with English, u can use https://translate.google.com/?hl=uk

File: src/Lesson11/Task5.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
+package Lesson11;
+
+public class Task5 {
```
Comment (Volodymyr95) [inline src/Lesson11/Task5.java]: What does it mean Task5?

File: src/Lesson11/Task5.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+                if (text.charAt(text.length() - 1 - i) != text.charAt(i)) {
+                    System.out.println("не є поліндромом");
+                    break;
+                }
+            }
+
+
+        }
+        if (k<=text.length()-k){
+            System.out.println("поліндром");
```
Comment (Volodymyr95) [inline src/Lesson11/Task5.java]: I 💓 but don't understand the Ukrainian language, if u have some problem with English, u can use https://translate.google.com/?hl=uk

File: src/Lesson11/Task5.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        }else {
+            System.out.println(858);
```
Comment (Volodymyr95) [inline src/Lesson11/Task5.java]: What does this number mean?  Magic number?

File: src/Lesson11/Task5.java [hunk 8]
```diff
@@ -0,0 +1,30 @@
+package Lesson11;
+
+public class Task5 {
+    String text;
```
Comment (Volodymyr95) [inline src/Lesson11/Task5.java]: Why do u declare this variable on the class layer if, u use it only in the method?

**Review by natalia-kozak — APPROVED** [review]

**Pull request merged**